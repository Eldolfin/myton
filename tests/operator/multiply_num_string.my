print(3 * "Hello, world! ") # error
