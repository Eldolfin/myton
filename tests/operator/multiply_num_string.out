[31merror[error]: unsupported operand type(s) for *: 'number' and 'str'
----- Traceback -----
  â”Œâ”€ <unknown>:1:9
  | 
1 | print(3 * "Hello, world! ") # error
  |          ^
[0m